WooCommerce/Classes/ViewRelated/Top Banner/TopBannerViewModel.swift

```diff
@@ -3,30 +3,41 @@ import UIKit
 /// View model for `TopBannerView`.
 ///
 struct TopBannerViewModel {
+
+    enum TopButtonType {
+        case chevron(handler: (() -> Void)?)
+        case dismiss(handler: (() -> Void)?)
```

Comment:
nit: it looks like we aren't using `dismiss` style (cross icon) anymore, maybe we can remove it for now. @Garance91540 do you know if we might be using a top banner that has "X" dismiss CTA at the top that isn't expandable anymore? (for example, in the old stats banners)

Reply:
I'll merge this, and will remove the `dismiss` type in a next PR if/when design confirms :-)